### PR TITLE
Adding release note and install procedure step for BZ 1767167

### PR DESCRIPTION
**Pull request by ousleyp**
@alexxa for your review :) 

Preview builds: 
- [Installation step](http://file.bos.redhat.com/pousley/103119/quotes-bug/cnv/cnv_install/installing-container-native-virtualization.html#cnv-deploying-cnv_installing-container-native-virtualization)
- [Known issue](http://file.bos.redhat.com/pousley/103119/quotes-bug/cnv/cnv_release_notes/cnv-release-notes.html#known-issues)

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1767167

### CODE REVIEW

**Comment by alexxa** [comment]
@ousleyp Looks good to me. @rthallisey agree?

**Review by bergerhoffer — COMMENTED** [review]
Just one suggestion, otherwise LGTM!

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -53,6 +53,25 @@ connectivity after being restarted. This issue is fixed in {CNVProductName} 2.1.
 
 == Known issues
 
+* When creating the *KubeVirt HyperConverged Cluster Operator Deployment*
+custom resource during {CNVProductName} installation, a YAML file is displayed.
```
[the remark this answers is not in the document]
Comment (ousleyp) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Definitely agree, great catch.

**Comment by rthallisey** [comment]
+1

**Pull request merged**